ui/app/pages/confirm-token-transaction-base/confirm-token-transaction-base.component.js

```diff
@@ -46,7 +46,7 @@ export default class ConfirmTokenTransactionBase extends Component {
   renderSubtitleComponent () {
     const { contractExchangeRate, tokenAmount } = this.props
 
-    const decimalEthValue = (tokenAmount * contractExchangeRate) || 0
+    const decimalEthValue = (tokenAmount * contractExchangeRate) || '0'
```

Comment:
This seems wrong :thinking: We're multuiplying a string here by `contractExchangeRate`